enzyme/test/Integration/ReverseMode/sret.cpp

```diff
@@ -34,5 +34,5 @@ int main() {
     dmyfunction(3,4,5,res);
     APPROX_EQ(res[0], 4*5., 1e-7);
     APPROX_EQ(res[1], 3*5., 1e-7);
-    APPROX_EQ(res[1], 3*4., 1e-7);
```

Comment:
Oh this is exceptionally weird, how was this passing before?

Reply:
It wasn't passing.